.circleci/config.yml

```diff
@@ -42,42 +42,36 @@ aliases:
     slack/notify:
       channel: $BUILD_NOTIFICATIONS_CHANNEL_ID
       custom: '{
-          "blocks": [
-            {
-              "type": "section",
-              "text": {
-                "type": "mrkdwn",
-                "text": "*Frontend v${PACKAGE_VERSION} is being prepared for release* :building_construction:"
-              }
-            },
-            {
-              "type": "section",
-              "text": {
-                "type": "mrkdwn",
-                "text": "A new release was created by ${CIRCLE_USERNAME}"
+            "blocks": [
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "*Frontend v${PACKAGE_VERSION} is being prepared for release* :building_construction:"
+                }
+              },
+              {
+                "type": "section",
+                "text": {
+                  "type": "mrkdwn",
+                  "text": "${SLACK_PARAM_MENTIONS} A new release was created by ${CIRCLE_USERNAME}"
+                }
               },
-              "fields": [
+              {
+                "type": "actions",
+                "elements": [
                   {
-                    "type": "mrkdwn",
-                    "text": "*Mentions*:\\n${SLACK_PARAM_MENTIONS}"
+                    "type": "button",
+                    "text": {
+                      "type": "plain_text",
```

Comment:
It's a shame that this is JSON embedded in YAML, as that's going to bypass most sorts of linting :(

Reply:
I will concur this. One day.